.github/workflows/shellcheck.yaml

```diff
@@ -8,4 +8,6 @@ jobs:
         uses: actions/checkout@v2
       - name: Run shellcheck
         id: shellcheck
-        uses: ludeeus/action-shellcheck@0.1.0
+        uses: ludeeus/action-shellcheck@1.1.0
+        env:
+          SHELLCHECK_OPTS: -e SC2012
```

Comment:
Please only ignore this on the affected line using the `# shellcheck disable=SC2012` annotation.